docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
+
+Choose appropriate resources depending on what kind of help you need:
+
+- Documentation sites
+  - Various information
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
```

Comment:
```suggestion
- **Documentation sites**
  - Various information
- **GitHub Discussions**
  - Questions
  - Unconfirmed bugs
  - Feature requests
  - Design discussions
- **GitHub Issues**
  - Confirmed bugs
- **Slack**
  - Instant messaging between contributors
- **ROS Discourse**
  - General topics that should be widely announced
```

Just making the help sources bold.

Reply:
Before
![image](https://user-images.githubusercontent.com/31987104/157447555-6f6c2ae6-46c3-4d16-b184-dc56903e3696.png)

After
![image](https://user-images.githubusercontent.com/31987104/157447533-468e8c4f-f9d7-48e7-aae2-45de451c877c.png)